eslint/babel-eslint-tests/test/helpers/verifyAndAssertMessages.js

```diff
@@ -31,7 +31,7 @@ export default function verifyAndAssertMessages(
           "../../../babel-eslint-shared-fixtures/config/babel.config.js",
         ),
       },
-      ...overrideConfig?.parserOptions,
+      ...(overrideConfig && overrideConfig.parserOptions),
```

Comment:
Now that we skip jest script transformer, should we add https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features/es-syntax.md to non-fixture test files?

Reply:
Oh thanks! I was looking for this rule but didn't remember the plugin name.

Reply:
I have enabled it, but support for `??` and `?.` hasn't been released yet (so it doesn't warn about them).